src/containers/TriggerTemplate/TriggerTemplate.js

```diff
@@ -0,0 +1,386 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import jsYaml from 'js-yaml';
+import './triggertemplate.scss';
+import { injectIntl } from 'react-intl';
+import {
+  DataTable,
+  InlineNotification,
+  Tab,
+  Tabs,
+  Tag
+} from 'carbon-components-react';
+import {
+  Table as DashboardTable,
+  FormattedDate
+} from '@tektoncd/dashboard-components';
+import {
+  getSelectedNamespace,
+  getTriggerTemplate,
+  getTriggerTemplatesErrorMessage,
+  isWebSocketConnected
+} from '../../reducers';
+
+import { fetchTriggerTemplate } from '../../actions/triggerTemplates';
+
+const {
+  Table,
+  TableBody,
+  TableContainer,
+  TableExpandedRow,
+  TableExpandRow,
+  TableExpandHeader,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow
+} = DataTable;
+
+const initialState = {
+  showYAML: false,
+  showDetails: false
+};
+
+export /* istanbul ignore next */ class TriggerTemplateContainer extends Component {
+  /* eslint-disable react/sort-comp */
+  constructor(props) {
+    super(props);
+    this.state = initialState;
```

Comment:
Alan requested a change here in my PR:
https://github.com/tektoncd/dashboard/pull/797#discussion_r353817901